pom.xml

```diff
@@ -37,6 +37,12 @@
             <version>${assertj-core.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jetbrains</groupId>
```

Comment:
just curious, why have you added it if it's not used?

Reply:
Actually it happened accidentally =(

Reply:
It will make PR cleaner, but as it's a test task, it doesn't really matter :)